src/main/java/cloud/eppo/configuration/ConfigurationBuffer.java

```diff
@@ -0,0 +1,116 @@
+package cloud.eppo.configuration;
+
+import cloud.eppo.ufc.dto.*;
+import cloud.eppo.ufc.dto.adapters.EppoModule;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Wrapper object passed between the various Configuration handling classes encapsulating flag and
+ * bandit configuration as well as whether the config is obfuscated.
+ */
+public class ConfigurationBuffer {
```

Comment:
nit: In other SDKs, this is called `Configuration`.

I think that's a simpler name (i.e., there's no buffering in `ConfigurationBuffer`). It's also a part of user-exposed API, so it's good to be consistent (unless there's a reason to differ)

Reply:
Agreed.  
Now that I've reoriented around the Configuration as the main source of truth and there's a builder, it's no longer buffering the config values from User => configStore or ConfigRequester => configStore